README.md

```diff
@@ -129,6 +129,80 @@ file { '/etc/tmpfiles.d/foo.conf':
 ~> Class['systemd::tmpfiles']
 ```
 
+### timer units
+Create a timer unit and a service unit to execute from
+that timer at the same time. The following will manage timer unit 
+service and enable it.
+
+```puppet
+systemd::timer{'runoften.timer':
+  timer_source   => "puppet:///modules/${module_name}/runoften.timer",
+  service_source => "puppet:///modules/${module_name}/runoften.service",
+  active         => true,
+  enable         => true,
+}
+```
+or a trivial daily run and manage the timer unit manually
+
+```puppet
+$_timer = @(EOT)
+[Timer]
+OnCalendar=daily
+RandomizedDelaySec=1d
+EOT
+
+$_service = @(EOT)
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/touch /tmp/file
+EOT
+
+if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
+  include systemd::systemctl::daemon_reload
```

Comment:
This will always end up included by the main class (which is included by unit_file).
https://github.com/camptocamp/puppet-systemd/blob/f81339ad10516905e460d2899432ce92d83a35d2/manifests/init.pp#L113